cmd/cluster-kube-apiserver-operator/render/render.go

```diff
@@ -0,0 +1,104 @@
+package render
+
+import (
+	"errors"
+
+	"github.com/golang/glog"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/assets"
+	"github.com/spf13/cobra"
+)
+
+type renderOpts struct {
+	assetDir             string
+	secretsDir           string
+	manifestTemplatesDir string
+	configDir            string
+	dataDir              string
+	cloudProviderDir     string
+}
+
+func NewRenderCommand() *cobra.Command {
+	renderOpts := &renderOpts{}
+	cmd := &cobra.Command{
+		Use:   "render",
+		Short: "Render kubernetes API server bootstrap manifests",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := renderOpts.Validate(); err != nil {
+				glog.Fatal(err)
+			}
+			if err := renderOpts.Run(); err != nil {
+				glog.Fatal(err)
+			}
+		},
+	}
+
+	cmd.Flags().StringVar(&renderOpts.assetDir, "asset-dir", "", "Output path for rendered assets.")
+	cmd.Flags().StringVar(&renderOpts.secretsDir, "secrets-dir", "", "A path to directory with certificates and secrets.")
+	cmd.Flags().StringVar(&renderOpts.manifestTemplatesDir, "manifest-templates-dir", "/manifest-templates", "A path to directory with manifest templates.")
+
+	cmd.Flags().StringVar(&renderOpts.configDir, "config-dir", "/etc/kubernetes/config", "A path to directory with configuration.")
```

Comment:
what is configuration?

Reply:
config.json ... kube api server config v1 I assume

Reply:
why a dir? Why not a yaml path?

Reply:
@sttts you need to mount it as a hostPath

Reply:
you can still mount a dir with the file inside. Would be clearer without implicit assumptions then.